### PR TITLE
Free trial extension: customize payment tasklist header

### PR DESCRIPTION
**Pull request by chihsuan**
Partially fixes https://github.com/Automattic/wc-calypso-bridge/issues/945

### WooCommerce Payments - tasklist

1. Header title - Change the copy from “It’s time to get paid” to “It’s time to test payments”
2. Header description - Change the copy from “[...] Verify your business details to start managing transactions with WooCommerce Payments” to “[...] Verify your business details to start testing transactions with WooCommerce Payments.”
3. Cta - Change copy from “Verify details” to “Test payments”.

### Payments task - tasklist

1. Header title - Change the copy from “It’s time to get paid” to “It’s time to test payments”
2. Header description - Change copy to "Give your customers an easy and convenient way to pay! Set up one (or more!) of our fast and secure online or in person payment methods."
3. Cta - Change copy from “Verify details” to “Test payments”.

## Test instructions

1. Make sure to copy over the latest changes in [woocmmerce-calypso-bridge-helper.php](https://gist.github.com/moon0326/cac46c70a2cee81b61faef517fef7178) and the plugin is activated.
2. Change [this line](https://gist.github.com/moon0326/cac46c70a2cee81b61faef517fef7178#file-a-woocommerce-calypso-bridge-helper-php-L33) to false in your local env. This forces your env to have free trial.
3. Go to WooCommerce > Home
4. Complete store details, add product task
5. Observe that the payment task list header matches the new copy
6. Install WCpay and choose a WCpay eligible  country
7. Go to WooCommerce > Home
8. Observe that the WCPay task list header matches the new copy

Payments task

WCPay task

### CODE REVIEW

**Review by ilyasfoo — COMMENTED** [review]
I just noticed in latest Figma there's additional T&C text added. However, I think the proper copy is `By clicking “Test payments”, you agree to the Terms of Service.`

1EGipzzulCg2uk4MsSf2x7-fi-14%3A5402

image

**Comment by chihsuan** [comment]
> I just noticed in latest Figma there's additional T&C text added. However, I think the proper copy is `By clicking “Test payments”, you agree to the Terms of Service.`

@ilyasfoo Thanks for checking the latest Figma. Changed in 55d9c2b44109dcbadf9e3e1d28926664ab3ae8e7.

Screenshot 2023-02-17 at 14 14 49

File: src/free-trial/fills/task-headers/woocommerce-payments.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+							</p>
+						</div>
+					</div>
+				);
+			} }
+		</WooOnboardingTaskListHeader>
+	);
+};
+
+registerPlugin( 'wc-calypso-bridge-woocommerce-payment-task-header', {
```
Comment (ilyasfoo) [inline src/free-trial/fills/task-headers/woocommerce-payments.js:85]:
```suggestion
registerPlugin( 'wc-calypso-bridge-woocommerce-payments-task-header', {
```

File: src/free-trial/fills/task-headers/payments.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+							</p>
+						</div>
+					</div>
+				);
+			} }
+		</WooOnboardingTaskListHeader>
+	);
+};
+
+registerPlugin( 'wc-calypso-bridge-payment-task-header', {
```
Comment (ilyasfoo) [inline src/free-trial/fills/task-headers/payments.js:64]:
```suggestion
registerPlugin( 'wc-calypso-bridge-payments-task-header', {
```

**Review by ilyasfoo — APPROVED** [review]
Awesome, tested well! LGTM! Super nitpick comment on standardizing the plugin names. 🚢

**Review by rjchow — APPROVED** [review]
This worked perfect with gutenberg disabled (due to the slotfill regression)! Thanks! 🚢

**Pull request merged**